article/_posts/2021-07-29-DiSCo.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: review
+title: "The Diffusion-Simulated Connectivity (DiSCo) Challenge"
+tags: tractography tractometry diffusion
+author: "Antoine Théberge"
+pdf: "http://hardi.epfl.ch/static/events/2021_challenge/"
+---
+
+# Introduction
+The Diffusion-Simulated Connectivity (DiSCo) Challenge is a connectivity estimation challenge presented at CDMRI, a workshop organized at MICCAI 2021. The goal of the challenge is to estimate the connectivity of a synthetic phantom composed of 16 regions of interests connected by interlacing fiber bundles. The participating teams must provide their own connectivity matrix and the team with the highest correlation coefficient with the ground-truth matrix will be declared winner.
```

Comment:
"regions of interests": "regions of interest"

Reply:
Indeed, fixed in d6aefbc0de10be4f97613718234cedae4bd20da6 and added acronym